executor/src/witgen/jit/quadratic_symbolic_expression.rs

```diff
@@ -406,6 +417,54 @@ impl<T: FieldElement, V: Ord + Clone + Hash + Eq + Display> QuadraticSymbolicExp
         };
         Some(Effect::RangeConstraint(solve_for.clone(), constraint))
     }
+
+    /// Tries to detect a range constraint of the form `(X - value1) * (X - value2) = 0`.
+    /// The most common example would be a bit constraint.
+    /// In that case, we can return a range constraint for `X` and mark the
+    /// constraint as complete.
+    // TODO: This function would not detect some equivalent constraints, like
```

Comment:
Does the docstring comment work properly if there is a regular comment between the docstring comment and the item it documents?

Reply:
Yes, looking good!

<img width="1004" alt="Screenshot 2025-04-22 at 16 48 56" src="https://github.com/user-attachments/assets/18c9dda1-50da-447b-9f93-05cbde2a0c24" />